package/src/frontend/browser/Agent.js

```diff
@@ -86,6 +86,44 @@ export const Agent = function(userAgent) {
      */
     matchesFacebookInAppBrowser: function() {
       return userAgent.match(/FBAN/) && userAgent.match(/FBAV/);
+    },
+
+    matchesDesktopChrome: function() {
+      return this.matchesChrome() && !this.matchesMobilePlatform();
+    },
+
+    matchesDesktopFirefox: function() {
+      return this.matchesFirefox() && !this.matchesMobilePlatform();
+    },
+    
+    matchesDesktopEdge: function() {
+      return this.matchesEdge() && !this.matchesMobilePlatform();
+    },
+    
+    /**
+   * Returns true on Google Chrome.
+   * @return {boolean}
+   */
+    matchesChrome: function() {
+      return matches(/Chrome\//i) &&
+             !matches(/Chromium/i);
+    },
```

Comment:
This will also be true in Edge. I think it's fine to support Chromium since it's pretty much the same browser?

Reply:
This will also be true in Edge.

Reply:
Why is this resolved?

Reply:
Forgot to add `!matches(/Edg/i)`. Just added that.